DEVELOPMENT.md

```diff
@@ -47,7 +47,7 @@ on to the next step.
 
 If you want to stop everything at any time, run `make down`.
 
-Note that `make dev` only runs the minimum amount of dependencies needed for things to work. If you'd like to run all the possible dependencies run `make alldeps` or directly the services available in the `docker-compose.yml` file. Keep in mind, though, that `make alldeps` does not start up Athens or Oympus, but **only** their dependencies.
```

Comment:
actually can you remove the word Olympus all together? I'd done a search-all on the word Olympus and obviously this one slipped by because it was a typo hah.

Reply:
FYI Olympus was a concept that was going to live along side Athens, but it's been removed for a while now :)

Reply:
Done! And thanks for the info 👍